From c99d69b014d1014079fe8b5a44e4861fd76435fa Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?= <avarab@gmail.com>
Date: Thu, 2 Sep 2010 19:13:28 +0000
Subject: [PATCH 23/25] builtin: use builtin.h in git-status
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Change wt-status.c to include gettext.h via builtin.h. This is needed
to be able to mark messages for translation.

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 wt-status.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/wt-status.c b/wt-status.c
index 54b6b03..7716dd9 100644
--- a/wt-status.c
+++ b/wt-status.c
@@ -1,3 +1,4 @@
+#include "builtin.h"
 #include "cache.h"
 #include "wt-status.h"
 #include "object.h"
-- 
1.7.2.2.614.g1dc9

